v22.2/upgrade-cockroach-version.md

```diff
@@ -124,8 +124,8 @@ By default, after all nodes are running the new version, the upgrade process wil
 
 When upgrading from {{ previous_version }} to {{ page.version.version }}, certain features and performance improvements will be enabled only after finalizing the upgrade, including but not limited to:
 
-- **SCRAM-SHA-256 authentication:** CockroachDB supports [SCRAM-SHA-256](security-reference/scram-authentication.html) authentication for clients in both CockroachDB Cloud and CockroachDB Self-Hosted. For SQL client sessions, it is now possible to use the authentication methods `password` (cleartext passwords), and `cert-password` (TLS client cert or cleartext password) with either CRDB-BCRYPT or SCRAM-SHA-256 stored credentials.
-- **Row-Level Time to Live (TTL):** CockroachDB has preview support for Time to Live ("TTL") expiration on table rows, also known as [Row-Level TTL](row-level-ttl.html). Row-Level TTL is a mechanism whereby rows from a table are considered "expired" and can be automatically deleted once those rows have been stored longer than a specified expiration time.
+- [Assume role authentication](use-cloud-storage-for-bulk-operations.html) allows you to limit the control access to your Amazon S3 buckets by creating IAM roles for users to assume. IAM roles do not have an association to a particular user. The role contains permissions that define the operations a user can complete. An IAM user can then assume a role to undertake bulk operations. As a result, the IAM user has access only to the assigned role, rather than having unlimited access to an S3 bucket. To use this feature, the entire cluster must be fully upgraded to v22.2.
```

Comment:
@chengxiong-ruan PTAL

Reply:
sorry, @mdlinville did you @ the wrong person? I'm not familiar with this heh.